quic/s2n-quic-core/src/crypto/tls.rs

```diff
@@ -130,6 +130,8 @@ pub trait Context<Crypto: crate::crypto::CryptoSuite> {
     //# peer's Finished message.
     fn on_handshake_complete(&mut self) -> Result<(), crate::transport::Error>;
 
+    /// Transfer application context from TLS connection to quic connection
+    fn on_application_context(&mut self, _context: Option<Box<dyn Any + Send + Sync>>) {}
```

Comment:
I think we can leave out the default impl here

Reply:
done